### PR TITLE
Add instructions about CRA2

### PR DESCRIPTION
**Pull request by stereobooster**
Issue:

## What I did

Updated documentation about CRA2

## How to test

No tests

### CODE REVIEW

**Comment by stereobooster** [comment]
Is there a way to directly reuse [CRA2 config](https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/create.js)?

File: MIGRATION.md [hunk 1]
```diff
@@ -170,6 +170,27 @@ Also make sure you have a `.babelrc` in your project directory. You probably alr
 }
 ```
 
+#### Use with CRA2
```
Comment (igor-dv) [inline MIGRATION.md]: please add a link to this from the table of content at the beginning of the file.
BTW, in v4.1 this is already not needed 😉

**Comment by ndelangen** [comment]
Since storybook 4.1 has been released with CRA2 support, does this PR still make sense? @igor-dv

**Review by igor-dv — APPROVED** [review]

**Comment by igor-dv** [comment]
BTW 4.1 is already released. I am not sure we need this already =)

**Comment by stereobooster** [comment]
I'm ok with closing it, if it works out of the box.

**Comment by igor-dv** [comment]
@ndelangen , sorry didn't notice your comment before 🤣

**Pull request closed without merging**